Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -893,6 +899,16 @@ extension BrowserViewController: TabManagerDelegate {
 
         tab.webView.removeFromSuperview()
     }
+
+    func isValidURL(url: NSURL) -> Bool {
```

Comment:
Nit: Let's name this `isWebPage` (about: and file:/// URLs are still valid URLs).